pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/retention/RetentionManagerTest.java

```diff
@@ -92,7 +92,7 @@ private void testDifferentTimeUnits(long pastTimeStamp, TimeUnit timeUnit, long
     conf.setRetentionControllerFrequencyInSeconds(0);
     conf.setDeletedSegmentsRetentionInDays(0);
     RetentionManager retentionManager = new RetentionManager(pinotHelixResourceManager, conf, controllerMetrics);
-    retentionManager.init();
+    retentionManager.start();
```

Comment:
I'm thinking whether the method `stop` be added at the end but it doesn't seem matter.

Reply:
I have the same feeling, but I'm just lazy lol